docs/index.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: v1
 entries:
   andi:
   - apiVersion: v1
-    created: 2019-11-21T10:56:05.248175-05:00
+    created: 2019-11-22T15:45:54.234216-05:00
```

Comment:
I think the created and digest ids should be reverted for andi, discovery-api, and forklift.

Keep kubernetes-data-platform